exercises/concept/coordinate-transformation/.docs/instructions.md

```diff
@@ -1,13 +1,16 @@
 # Instructions
 
-Your design company has primarily been working with CSS transformations to build web pages. After some discussion, a decision is made
-to start using JavaScript to perform some calculations dynamically. Some of your teammates are less experienced with JavaScript,
-so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.
+Your design company has primarily been working with CSS transformations to build web pages. After some discussion, a
+decision is made to start using JavaScript to perform some calculations dynamically. Some of your teammates are less experienced with
+JavaScript, so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.
```

Comment:
I meant this: https://exercism.org/docs/building/markdown/markdown

```suggestion
Your design company has primarily been working with CSS transformations to build web pages.
After some discussion, a decision is made to start using JavaScript to perform some calculations dynamically.
Some of your teammates are less experienced with JavaScript, so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.
```

etc ;)

Reply:
I see.
I feel like I managed this time around.